src/components/ChartContextMenu.svelte

```diff
@@ -48,6 +49,44 @@
 
   const dateFormatter = timeFormat('%Y-%m-%d');
   const timeFormatter = timeFormat('%H:%M:%S');
+
+  const getTelemetryPath = () => {
+    if ($store.probe.type === 'histogram') {
+      return ['main', 'parent'].includes($store.productDimensions.process)
+        ? `payload.histograms.${$store.probe.name}`
+        : `payload.processes.${$store.productDimensions.process}.histograms.${$store.probe.name}`;
+    }
+    return undefined;
+  };
+
+  const table =
+    $store.productDimensions.channel === 'nightly'
+      ? 'main_nightly'
+      : 'main_1pct';
+
+  const REDASH_PROBE_COMPARISON_URL =
+    'https://sql.telemetry.mozilla.org/queries/82247/source?';
+
+  const getComparisonViewinSTMO = (clicked, hovered) => {
+    let queryParams = new URLSearchParams();
```

Comment:
According to the [URLSearchParams examples on MDN](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#examples), you should be able to create them all in one go. E.g.:

```suggestion
    const queryParams = new URLSearchParams({ p_Table: `telemetry.${table}`, p_Probe: getTelemetryPath(), ...});
```

Reply:
Ah yes, this is a nicer way to construct it. Thanks @wlach!